RentItAll/src/items/ControllerServlet.java

```diff
@@ -0,0 +1,120 @@
+package items;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+import javax.servlet.GenericServlet;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+public class ControllerServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+    private ItemDBC itemDBC;
+    
+    public void init() {
+        String jdbcURL = getServletContext().getInitParameter("jdbcURL");
+        String jdbcUsername = getServletContext().getInitParameter("jdbcUsername");
+        String jdbcPassword = getServletContext().getInitParameter("jdbcPassword");
+ 
+        itemDBC = new ItemDBC(jdbcURL, jdbcUsername, jdbcPassword); 
+    }
+
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        doGet(request, response);
+    }
+    
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String action = request.getServletPath();
```

Comment:
Create a session variable on your JSP instead and just get the value from that page.